webhook-app/github_helper.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Helpers for interacting with the github API and hook events."""
+
+import os
+
+import github3
+
+
+def github_user():
+    """Returns the bot's username."""
+    return os.environ['GITHUB_USER']
+
+
+def get_client():
+    """Returns an authenticated github3 client."""
+    return github3.login(
+        github_user(), os.environ['GITHUB_ACCESS_TOKEN'])
```

Comment:
Nice! Environment variables seem like a good way to handle the secrets (12-factor app and all that jazz). I'm sure I'll see how they are populated later. Nice to see they don't have to go in `app.yaml`.